packages/@roots/bud/src/interface.ts

```diff
@@ -60,37 +55,29 @@ declare module '@roots/bud-framework' {
       noEmit: boolean
       manifest: boolean
       minify: boolean
-      mode: 'production' | 'development'
+      mode: ('production' | 'development') &
+        Webpack.Configuration['mode']
       namedModules: boolean
-      node: {
-        module: string
-        dns: string
-        fs: string
-        http2: string
-        net: string
-        tls: string
-        child_process: string
-      }
-      parallelism: number
-      profile: boolean
-      runtimeChunkEnabled: boolean
+      node: Webpack.Configuration['node']
+      parallelism: Webpack.Configuration['parallelism']
+      profile: Webpack.Configuration['profile']
       runtimeChunk: Webpack.Configuration['optimization']['runtimeChunk']
-      splitChunksEnabled: boolean
-      splitChunks: {
-        chunks: string
-        minSize: number
-        maxSize: number
-        minChunks: number
-        maxAsyncRequests: number
-        maxInitialRequests: number
-      }
-      stats: boolean
-      target: string
-      resolve: {
-        alias: {[key: string]: any}
-        extensions: string[]
-        modules: string[]
-      }
+      splitChunks: Webpack.Configuration['optimization']['splitChunks']
+      stats: Webpack.Configuration['stats']
+      target: Webpack.Configuration['target']
+      resolve:
+        | Webpack.Configuration['resolve']
```

Comment:
It's too crazy supporting that much nesting. It's better to just leave people with the top-level `Webpack.Configuration['splitChunks']` key to configure for themselves, if they disagree with defaults.